shaders/particles/simulate.comp

```diff
@@ -52,7 +52,7 @@ void main()
         ParticleInstance instance;
         instance.position = particle.position;
         instance.materialIndex = particle.materialIndex;
-        instance.size = particle.size;
+        instance.size = particle.size.xy;
```

Comment:
why change them to vec3's if you only use the vec2 part?

Reply:
there's no implementation for it yet but I plan to add size scaling. just like the rotation scaling, they can shrink overtime for example. The third float is planned to be used as a 'velocity' for the scaling.

Not sure why I put this in this PR though xd I forgot about it halfway I think. Implementing functionality for this is added to a task in codecks though